keyboards/kudox/readme.md

```diff
@@ -5,25 +5,25 @@
 </p>
 
 <p align="center">
-<img src="https://raw.githubusercontent.com/kumaokobo/kudox-keyboard/master/img/kudox-pcb.jpg" alt="Kudox PCB rev2.0" width="600"/>
+<img src="https://raw.githubusercontent.com/kumaokobo/kudox-keyboard/master/img/kudox-rev3-image1.jpg" alt="Kudox rev3.0" width="600"/>
 </p>
 
 - Keyboard Maintainer: [Kumao Kobo](https://github.com/kumaokobo)  
-- Hardware Supported: Kudox PCB rev1.0 rev2.0 w/ Pro Micro  
+- Hardware Supported: Kudox PCB rev1.0 rev2.0 rev3.0 w/ Pro Micro  
 
 Make example for this keyboard (after setting up your build environment):
 
 ```sh
-make kudox/rev2:default
+make kudox/rev3:default
 ```
 
 
 Example of flashing this keyboard:
 
 ```sh
-make kudox/rev2:default:flash
+make kudox/rev3:default:flash
 ```
 
-*keymaps/default is for rev2.0. If you want to use rev1.0, you should remove Left-06 key and Right-06 key.*
+*keymaps/default is for rev3.0. If you want to use rev1.0, you should remove Left-06 key and Right-06 key.*
 
```

Comment:
This note isn't needed. Because each revision has its own `keymaps` directory, the default keymap therein is specific to that revision and can't be compiled for the other revisions.

```suggestion
```